k8s/aat/common/divorce/div-cos.yaml

```diff
@@ -41,12 +41,22 @@ spec:
         SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE: false
         SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_ENABLED: true
         BULKACTION_MINCASES: 5
+        DATAEXTRACTION_STATUS_DA_EMAILFROM: noreply@mail-divorce-nonprod.platform.hmcts.net
         DATAEXTRACTION_STATUS_DA_EMAILTO: da_data_extraction@sharklasers.com
         DATAEXTRACTION_STATUS_DN_EMAILTO: dn_data_extraction@sharklasers.com
         DATAEXTRACTION_STATUS_AOS_EMAILTO: aos_data_extraction@sharklasers.com
         SCHEDULER_ENABLED: true
         SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_CRON: "0 0 4 ? * * *"
         SCHEDULER_MAKE_ELIGIBLE_CASES_AOS_OVERDUE_ENABLED: true
+        SPRING_MAIL_HOST: smtp.sendgrid.net
+        SPRING_MAIL_PORT: 587
+        SPRING_MAIL_PROPERTIES_MAIL_SMTP_SSL_TRUST: smtp.sendgrid.net
+      keyVaults:
+        "div":
+          resourceGroup: div
+          secrets:
+          - spring-mail-username
```

Comment:
you need to include all secrets from the base chart and then append spring mail password

https://github.com/hmcts/div-case-orchestration-service/blob/master/charts/div-cos/values.yaml#L34-L48

Reply:
added